litex/tools/litex_soc_gen.py

```diff
@@ -62,56 +63,66 @@ def build(self, fragment, build_dir, build_name, **kwargs):
 # LiteX SoC Generator ------------------------------------------------------------------------------
 
 class LiteXSoCGenerator(SoCMini):
-    def __init__(self, name="litex_soc", sys_clk_freq=int(50e6), **kwargs):
+    def __init__(self, name="litex_soc", sys_clk_freq=int(50e6), n_mi=3, n_si=0, **kwargs):
+        assert (n_mi + n_si) < 5
```

Comment:
Is this assert arbitrary? If yes, it's probably better to remove it, if not it's better to put a comment.

Reply:
I set the "assert" as I was thinking about a limitation in the number of interfaces (4 interfaces per standalone SoC.).